core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -15,6 +15,13 @@
 
     <div ng-repeat="question in data.questionList.getQuestions() track by $index">
       <anchor-header header-id="question.getId()" header-text="Question <[$index +1]>"></anchor-header>
+      <anchor-header header-id="getSubfieldId(question,'Multiple Choice')"
+                     header-text="<[getQuestionTypeLabel(question.getInteractionId())]> &#x25BC;" ng-click="chooseQuestionType = true; $parent.modalWrapperVisible = true;">
+      </anchor-header>
+
+      <div ng-show="chooseQuestionType && $parent.modalWrapperVisible" class="question-type-wrapper">
+        <button type="button" class="oppia-question-button btn btn-primary" ng-click="changeQuestion('MultipleChoiceInput', $index); chooseQuestionType = false; $parent.modalWrapperVisible= false;">Multiple Choice Input</button>
```

Comment:
The problem with setting variables like this is that is breaks abstraction barriers, which will cause problems down the road when there is a refactoring operation or if there is a need to debug. Having more verbose code is the cost for setting up abstraction barriers, but the benefits outweigh the costs in this case.